README.md

```diff
@@ -1,20 +1,31 @@
 # commercetools-adyen-integration
 [![Build Status](https://github.com/commercetools/commercetools-adyen-integration/workflows/CI/badge.svg?branch=master)](https://github.com/commercetools/commercetools-adyen-integration/actions)
 
-`commercetools-adyen-integration` provides an [Adyen Web Components](https://docs.adyen.com/checkout/components-web) based integration between the commercetools and Adyen PSP. Integration supports all the payment methods available as an [Adyen Web Component](https://docs.adyen.com/checkout/components-web). For a full list of all supported web component based payment methods please refer to [supported payment methods](https://docs.adyen.com/checkout/supported-payment-methods).
-
-> Note: since the integration relies on the usage of Adyen's web components it does not need to process sensitive credit card data and thus is fully PCI DSS **compliant**.
+`commercetools-adyen-integration` provides an [Adyen Web Components](https://docs.adyen.com/checkout/components-web) based integration between the commercetools and Adyen PSP.
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
+
+- [Features](#features)
 - [Overview](#overview)
 - [Extension module](#extension-module)
 - [Notification module](#notification-module)
 - [Contribution Guide](#contribution-guide)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Features
```

Comment:
Maybe rename it to `Supported features` then we can omit below the repetitive `Integration supports`?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/863/commits/6b36b219fa6682dd2f95dcc12e0b056360bfd2b4